src/react/2-intro-to-jsx.md

```diff
@@ -94,9 +94,9 @@ First, notice how instead of using a string to store the styles (like `style=""`
 > />
 > ```
 
-Inside of `style={}` we have an object of styles whose keys are css attributes and values are css values. This structure closely resembles the [`CSSStyleDeclaration`](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration) object used by the browser to store element styles.
+The object passed into `style={}` should be an object whose keys are css attributes and values are css values. This structure closely resembles the [`CSSStyleDeclaration`](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration) object used by the browser to store element styles.
```

Comment:
The first sentence sounds overly wordy: "the object should be an object", "css attributes, css values".

Reply:
Agreed, I changed it back. It came out of a convo I had with Justin but I think components makes more sense here.

Reply:
Agreed, I changed it to make it less awkward.